controllers/managedmcg_controller.go

```diff
@@ -319,6 +323,37 @@ func (r *ManagedMCGReconciler) reconcileNoobaaComponent() error {
 	return err
 }
 
+func (r *ManagedMCGReconciler) reconcileOCSCSV() error {
+	var csv *opv1a1.ClusterServiceVersion
+	var err error
+	if csv, err = r.getCSVByPrefix("ocs-operator"); err != nil {
+		return err
+	}
+	var isChanged bool
+	deployments := csv.Spec.InstallStrategy.StrategySpec.DeploymentSpecs
+	for i := range deployments {
+		name := deployments[i].Name
+		switch name {
+		case "ocs-operator", "ocs-metrics-exporter", "rook-ceph-operator":
+			zero := int32(0)
+			replicaCount := &deployments[i].Spec.Replicas
+			if **replicaCount != zero {
+				r.Log.Info("Downscaling deployment replicas", "OCS Deployment", name)
+				*replicaCount = &zero
+				isChanged = true
+			}
+		default:
+			r.Log.Info("Could not find deployment", "Deployment", name)
+		}
+	}
+	if isChanged {
+		if err := r.update(csv); err != nil {
+			r.Log.Error(err, "Failed to update OCS CSV")
```

Comment:
do we need to return an error for this condition?

Reply:
Sounds good, adding.